src/Registration/cReg/include/sirf/Reg/NiftiImageData.h

```diff
@@ -623,7 +623,7 @@ class NiftiImageData : public ImageData
     {
 	return new NiftiImageData(*this);
     }
-    virtual ObjectHandle<DataContainer>* new_data_container_handle() const
+    virtual ObjectHandle<DataContainer>* new_data_container_handle(const bool initialise_with_0 = false) const
```

Comment:
as I indicated previously (I believe), we'd really have to fill with 0 if `initialise_with_0=true`

Reply:
done